kalite/distributed/static/js/distributed/exercises.js

```diff
@@ -222,7 +222,12 @@ window.AttemptLogCollection = Backbone.Collection.extend({
     },
 
     url: function() {
-        return "/api/attemptlog/?" + $.param(this.filters, true);
+        if (window.statusModel.get("is_logged_in")) {
+            return "/api/attemptlog/?" + $.param(this.filters, true);
+        }
+        else {
+            window.location = "/securesync/login";
+        }
```

Comment:
Also, I'm curious: Did you test this? I don't know how `is_logged_in` would be getting updated within a page load...